packages/vaex-core/vaex/dataframe.py

```diff
@@ -384,6 +384,7 @@ def apply(self, f, arguments=None, dtype=None, delay=False, vectorize=False):
 
         :param f: The function to be applied
         :param arguments: List of arguments to be passed on to the function f.
+        :param bool multiprocessing: Use multiple processes to avoid the GIL (Global interpreter lock)
```

Comment:
Should we take this opportunity to document the rest of the kwargs here?

Reply:
yeah, vectorize at least